dashboards-observability/public/components/event_analytics/utils/utils.tsx

```diff
@@ -381,8 +381,12 @@ export const getPropName = (queriedVizObj: {
   name: string;
   label: string;
 }) => {
-  if (queriedVizObj[CUSTOM_LABEL] === '' || queriedVizObj[CUSTOM_LABEL] === undefined) {
-    return `${queriedVizObj.aggregation}(${queriedVizObj.name})`;
+  if (queriedVizObj) {
```

Comment:
Can we take look at reviews from previous PRs and having one more reviewer internally to review to avoid spending effort reviewing the same issues? Previously we've reviewed some similar code already like this, below code piece can be further simplified as

```
if (queriedVizObj && !isEmpty(queriedVizObj[CUSTOM_LABEL]))  return `${queriedVizObj.aggregation}(${queriedVizObj.name})`;
return '';
```

Reply:
@mengweieric Sure, will ensure in next PRs that such small issues can be addressed internally.